src/reflect/scala/reflect/internal/Mirrors.scala

```diff
@@ -197,29 +197,35 @@ trait Mirrors extends api.Mirrors {
         case _                                                   => MissingRequirementError.notFound("package " + fullname)
       }
 
-    @deprecated("Use overload that accepts a String.", "2.13.0")
+    @deprecated("use overload that accepts a String.", since = "2.13.0")
```

Comment:
somebody took notes in class.

Reply:
I'm surprised you didn't comment on the anachronism.

Reply:
Yes, it's always a wonder I didn't comment, but every once in a while I exercise uncommon restraint.

Reply:
BTW, the word for this is prolepsis, it occurs to me. And there's also a word for that. https://en.wikipedia.org/wiki/L%27esprit_de_l%27escalier